pkg/tlsconfig/config.go

```diff
@@ -24,21 +24,39 @@ type Options struct {
 	KeyFile            string
 }
 
-// Default is a secure-enough TLS configuration.
-var Default = tls.Config{
+// Extra (server-side) accepted CBC cipher suites - will phase out in the future
+var acceptedCBCCiphers = []uint16{
+	tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
+	tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
+	tls.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
```

Comment:
I don't believe there is any benefit of RSA over ECDSA here... and ECDSA is much faster, so perhaps move this up in the suite?